__tests__/core/runner.test.ts

```diff
@@ -692,13 +692,10 @@ describe('runner', () => {
   });
 
   /**
-   * TODO: Move this as part of integration test
    * Its really hard to ensure the journey/end is called for a real world page
    * without actually testing on a real world webpage.
-   * 
-   * FLAKY skipped
    */
-  it.skip('run - ensure journey/end is written for real world pages', async () => {
+  it('run - ensure journey/end is written for real world pages', async () => {
```

Comment:
We cant get a better indicator than this test, I am keeping this as it is. 

I did not want to test the sonic-boom package functionality in our agent.